src/multipole_header.F90

```diff
@@ -0,0 +1,152 @@
+module multipole_header
+
+  implicit none
+
+  !========================================================================
+  ! Multipole related constants
+
+  ! Formalisms
+  integer, parameter :: FORM_MLBW = 2, &
+                        FORM_RM   = 3, &
+                        FORM_LRM  = 7
```

Comment:
ENDF-102 calls it "R-Matrix Limited" (RML). Probably best to stay consistent with that.